api/routes/scripts.py

```diff
@@ -1,21 +1,30 @@
-from flask import jsonify
+import logging
 from flask import request
 import hashlib
 import json
+from logger.logger import get_logger
+
+api_routes_logger = get_logger("api_routes_loger", "api_routes.log")
```

Comment:
Infinite log file? Waste of server memory.

In your new Logger class (which you will create of course) add some logic to create new log file when it's capacity threshold is exceeded (let's say 20Mb or something) and also remove old log files (we don't want to store a million of files).

Ask GPT about the best approach. Maybe some library already does everything you need